lib/web/scripts/node-join/install.sh

```diff
@@ -15,9 +16,11 @@ MACOS_STDERR_LOG="/var/log/teleport-stderr.log"
 MACOS_STDOUT_LOG="/var/log/teleport-stdout.log"
 SYSTEMD_UNIT_PATH="/lib/systemd/system/teleport.service"
 TARGET_PORT_DEFAULT=443
+TELEPORT_APP_BUNDLES_darwin="tctl.app tsh.app"
```

Comment:
```suggestion
TELEPORT_APP_BUNDLES_darwin=(tctl.app tsh.app)
```

This is treated like an array but not declared as one